gammapy/background/reflected.py

```diff
@@ -5,24 +5,69 @@
 import numpy as np
 from numpy import sin, cos, arctan2, sqrt
 from ..image import exclusion_distance, lookup
+from astropy.table import Table
 
 __all__ = [
     'ReflectedRegionMaker',
-    'ReflectedBgMaker',
+    'CircularOffRegions',
+    'find_reflected_regions_for_event_list'
 ]
 
 
+class CircularOffRegions(Table):
+    """List of circular OFF regions
+
+    Parameters
+    ----------
+    data_rows : `~np.array`
+        Array containing the circle definitions: x, y, radius
+    """
+
+    @property
+    def number_of_regions(self):
+        """Number of OFF regions"""
+        return len(self)
+    
+    def info(self):
+        """Summary info string."""
+        s = super(CircularOffRegions, self).__repr__()
+        return s
+
+    def to_ds9(self, filename):
+        """Write ds9 regions file"""
+
+        fmt = 'fk5; circle({x},{y},{r})\n'
```

Comment:
I think it's not yet completely clear if your CircularOffRegions work in pixel or sky coordinates.
x, y suggests pixel coordinates, if it's sky coordinates, please use lon, lat.
And here you hard-code "fk5" which is for RADEC, so your current code cannot work with images in GALACTIC coordinates?

In the end the reflected region finding (rotating and checking for overlap with the mask via the distance image) should be done in pixel coordinates. The end user probably wants to input sky coordinates.

Now that I've thought about it a bit, I think the best approach here would be to use [SkyCircularAperture](http://photutils.readthedocs.org/en/latest/api/photutils.aperture_core.SkyCircularAperture.html) and [CircularAperture](http://photutils.readthedocs.org/en/latest/api/photutils.aperture_core.CircularAperture.html) from photutils (because they are the appropriate objects to represent regions and already have nice functionality to convert back and forth between pixel and sky coordinates via a wcs object) and only to add a `RegionList` class (or even use a plain Python list of apertures and pass that between find_regions and select_events).

Do you want to try this out or just stick with what you have now?